__tests__/integration.test.js

```diff
@@ -55,3 +56,27 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("responds with a 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  })
+  test("responds with an article object, with all of the correct properties", () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+      console.log(body.article)
+        body.article.forEach((property) => {
+          expect(typeof property.article_id).toBe("number");
```

Comment:
You need to check the number is the one you've requested, in this case 1. 

At the moment you could get back any article and this would pass.